When a startup gets an error

```
16:48:59 [ERROR] [MiddlewareCantBeLoaded] An unknown error was thrown while loading your middleware.
  Error reference:
    https://docs.astro.build/en/reference/errors/middleware-cant-be-loaded/
  Stack trace:
    at /Users/sgalcheung/....../mkhismkh/node_modules/.pnpm/astro@5.5.2_@types+node@22.13.10_jiti@1.21.7_rollup@4.35.0_typescript@5.8.2_yaml@2.7.0/node_modules/astro/dist/core/middleware/loadMiddleware.js:8:24
    [...] See full stack trace in the browser, or rerun with --verbose.
  Caused by:
  Cannot read properties of undefined (reading 'split')
    at eval (eval at runInlinedModule (file:///Users/sgalcheung/....../mkhismkh/node_modules/.pnpm/vite@6.2.2_@types+node@22.13.10_jiti@1.21.7_yaml@2.7.0/node_modules/vite/dist/node/module-runner.js:1049:11), <anonymous>:7:68)
    at async SSRCompatModuleRunner.directRequest (file:///Users/sgalcheung/....../mkhismkh/node_modules/.pnpm/vite@6.2.2_@types+node@22.13.10_jiti@1.21.7_yaml@2.7.0/node_modules/vite/dist/node/module-runner.js:1271:61)
    at async SSRCompatModuleRunner.cachedRequest (file:///Users/sgalcheung/....../mkhismkh/node_modules/.pnpm/vite@6.2.2_@types+node@22.13.10_jiti@1.21.7_yaml@2.7.0/node_modules/vite/dist/node/module-runner.js:1167:76)
    at async ESModulesEvaluator.runInlinedModule (file:///Users/sgalcheung/....../mkhismkh/node_modules/.pnpm/vite@6.2.2_@types+node@22.13.10_jiti@1.21.7_yaml@2.7.0/node_modules/vite/dist/node/module-runner.js:1049:5)
    at async SSRCompatModuleRunner.directRequest (file:///Users/sgalcheung/....../mkhismkh/node_modules/.pnpm/vite@6.2.2_@types+node@22.13.10_jiti@1.21.7_yaml@2.7.0/node_modules/vite/dist/node/chunks/dep-B0fRCRkQ.js:30915:23)

```

--- a/README.md
+++ b/README.md
@@ -61,7 +61,7 @@ This project uses the following technologies:
    # Clerk
    PUBLIC_CLERK_PUBLISHABLE_KEY=your_clerk_publishable_key
    CLERK_SECRET_KEY=your_clerk_secret_key
-   ADMIN_USER_IDs=comma_separated_admin_user_ids
+   ADMIN_USER_IDS=comma_separated_admin_user_ids
    ```
 
 4. Start the development server:

--- a/README.md
+++ b/README.md
@@ -84,7 +84,7 @@ This project uses the following technologies:
 1. Create a new Clerk application at [clerk.com](https://clerk.com)
 2. Configure authentication methods (email, social logins, etc.)
 3. Copy your publishable key and secret key to the `.env` file
-4. Add admin user IDs to the `ADMIN_USER_IDs` environment variable
+4. Add admin user IDs to the `ADMIN_USER_IDS` environment variable
 
 ### Deployment
 

--- a/src/env.d.ts
+++ b/src/env.d.ts
@@ -5,7 +5,7 @@ interface ImportMetaEnv {
   readonly SUPABASE_KEY: string;
   readonly PUBLIC_CLERK_PUBLISHABLE_KEY: string;
   readonly CLERK_SECRET_KEY: string;
-  readonly ADMIN_USER_IDs: string;
+  readonly ADMIN_USER_IDS: string;
 }
 
 interface ImportMeta {

